apps/frontend/app/(client)/(main)/settings/_components/context.ts

```diff
@@ -0,0 +1,56 @@
+import type { SettingsFormat } from '@/types/type'
+import { createContext, useContext } from 'react'
+import type { FieldErrors, UseFormRegister } from 'react-hook-form'
+
+export interface GetProfile {
```

Comment:
```suggestion
export interface Profile {
```
객체 타입이라서 동사보다는 명사형으로 이름 짓는 건 어떨까요?